### DISCUSSION TITLE
Annotation based mutation sample in the docs

### DISCUSSION THREAD

**Discussion by wondering639** [ChilliCream/hotchocolate#4705] [Q&A] [answered]
The example labeled Annotation-based at https://chillicream.com/docs/hotchocolate/defining-a-schema/mutations#usage doesn't have an annotation or I don't understand it:
![image](https://user-images.githubusercontent.com/25722498/151674512-98c7ca50-414e-48e7-ad6c-7694026cc764.png)
Can someone please explain? Thanks!

**Comment by tobias-tengler** [comment, marked as answer]
While "Annotation-based" implies the usage of an attribute, in this example it only means we are not using Code-first APIs such as a `descriptor` and `ObjectType`. In general you can think of "Annotation-based" as "I write regular classes/records and sprinkle in some Hot Chocolate functionality using attributes when necessary".

To explain the example: The class "Mutation" represents a GraphQL object type of the same name with a field called "addBook". The method called "AddBook" is what computes the value for said field.
